modules/modules.mk

```diff
@@ -50,8 +62,8 @@ endif
 # The master list of all moose modules
 MODULE_NAMES := "chemical_reactions contact fluid_properties functional_expansion_tools heat_conduction level_set misc navier_stokes phase_field porous_flow rdg richards solid_mechanics stochastic_tools tensor_mechanics xfem external_petsc_solver"
 
-###############################################################################
-########################## MODULE REGISTRATION ################################
+################################################################################
+########################## MODULE REGISTRATION #################################
 ###############################################################################
```

Comment:
Either you wanted to add the `#` s and forgot the 3rd line or you did not want to add them :-)

Reply:
Oh damn... one comment character.

Reply:
I do want to add these to get it out to 80. Since the comment in the block above also goes out to 80.